pkg/controller/plan/migration.go

```diff
@@ -1125,7 +1125,8 @@ func (r *Migration) execute(vm *plan.VMStatus) (err error) {
 		}
 
 		// only for OVA, fetch config from the conversion pod
-		if r.Source.Provider.Type() == v1beta1.Ova {
+		source := r.Source.Provider.Type()
+		if source == v1beta1.Ova || source == v1beta1.VSphere {
```

Comment:
please update the comment above
and how about using switch-case? it would be a bit more elegant in my opinion